invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -13,49 +13,71 @@
 
 <div class="ui container fluid page-subheader-outer with-submenu rel-pt-2">
   <div class="ui container relaxed grid page-subheader">
-    <div class="row">
-      <div class="two wide column">
+    <div class="row pb-0">
+      <div class="sixteen wide mobile five wide tablet four wide computer column">
         {% if logo %}
-        <div class="ui rounded image medium community-image">
-          <img src="{{logo.links.self}}" alt="No image found"/>
+        <div class="ui rounded fluid image community-image rel-mb-2">
+          <img src="{{logo.links.self}}" alt=""/>
         </div>
         {% else %}
-        <div class="ui rounded image">
-          <img src="/static/images/square-placeholder.png" alt="No image found"/>
+        <div class="ui rounded fluid image rel-mb-2">
+          <img src="/static/images/square-placeholder.png" alt=""/>
         </div>
         {% endif %}
       </div>
-      <div class="ten wide column">
-          <h2 class="ui header">{{ community.metadata.title }}</h2>
+
+      <div class="sixteen wide mobile eleven wide tablet twelve wide computer column rel-mb-2">
+        <h2 class="ui header">{{ community.metadata.title }}</h2>
+        <div class="rel-mb-2">
           {{ truncate_text(community.metadata.description) }}
-          {% if community.metadata.website %}
-            <a href="{{community.metadata.website}}"> <i class="linkify icon"></i>
-              {{community.metadata.website}}
-            </a>
-          {% endif %}
-          {% if community.metadata.type %}
-            <i class="tag icon community-tags"></i>
-            <div class="ui label small grey type-label">
-              {{ types[community.metadata.type] }}
+        </div>
+
+        <div class="ui grid">
+          <div class="row pb-0">
+            <div class="sixteen wide mobile ten wide tablet ten wide computer column">
+              {% if community.metadata.type %}
+              <div class="mt-5">
+                <i class="tag icon community-tags" aria-hidden="true"></i>
```

Comment:
```suggestion
                <i class="tag icon" aria-hidden="true"></i>
```
I think the community-tags class was removed from CSS, to be checked